assets/styles/_variables.scss

```diff
@@ -26,5 +26,33 @@ $color-background-light: #f2f6fe;
 $color-background-dark: #091a41;
 $color-background-hero: $color-brand;
 
+:root {
+  // editable colors for WP Customizer
+  --color-brand: #{$color-brand};
+  --color-accent: #{$color-accent};
+  --color-accent-dark: #{darken($color-accent, 10%)};
```

Comment:
This variable is only used once from what i see in `.themeReset`, I don't think it should be part of editable colours, it seems connected to `color-accent` so it should change accordingly with customizer changing `color-accent`. Is this possible?

Reply:
**--color-accent-dark** is a link `hover` color. I tried to generate darken color based on **--color-accent** but scss function doesn't work with css variables. The only way I found is using color in hsl, using hue values as variable and change lightness to make it darker:
```
--color-accent-hue: 20;
--color-accent: hsl(var(--color-accent-hue),  95%,  **53%**);
--color-accent-dark: hsl(var(--color-accent-hue),  95%,  **43%**);
```
but ie11 polyfill doesn't work with such solution :(
or maybe we can change link hover color to `--color-background-dark` to make it consistent with button link and remove --color-accent-dark at all
![image](https://user-images.githubusercontent.com/7567926/84127552-b84df600-aa3f-11ea-9582-c5fe26c21dcb.png)